xcflushd.gemspec

```diff
@@ -22,15 +26,15 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.1.0'
 
-  spec.add_runtime_dependency "3scale_client", "~> 2.10.0"
+  spec.add_runtime_dependency "3scale_client", "~> 2.10.0", ">= 2.10.0"
```

Comment:
`gem 2.6.4` says:
```
WARNING:  pessimistic dependency on 3scale_client (~> 2.10.0) may be overly strict
  if 3scale_client is semantically versioned, use:
    add_runtime_dependency '3scale_client', '~> 2.10', '>= 2.10.0'
```
My understanding is that this will enable `2.11.0` to be used whenever it comes out (and we unlock it), but not `3.0.0`.